example/package-lock.json

```diff
@@ -90,9 +90,7 @@
         "@as-pect/assembly": "^4.0.0",
         "@as-pect/cli": "^4.0.0",
         "@as-pect/core": "^4.0.0",
-        "@assemblyscript/loader": "^0.13.1",
-        "as-bignum": "^0.1.7",
-        "assemblyscript": "^0.13.8"
```

Comment:
why the assembly script and its loader are deleted?

Reply:
* Updated